.github/workflows/ci.yml

```diff
@@ -37,3 +37,24 @@ jobs:
           platforms: amd64,arm64,ppc64le
       - name: Build and Test Docker Image
         run: docker/build.sh
+      - name: Set up Helm
+        uses: azure/setup-helm@v3
+        with:
+          version: v3.12.1
+      - uses: actions/setup-python@v4
+        with:
+          python-version: '3.10'
+          check-latest: true
+      - name: Lint chart
+        run: helm lint --strict helm/trino-gateway
+      - name: Create kind cluster
+        uses: helm/kind-action@v1.10.0
+        with:
+          cluster_name: tgw
+      - name: load images
+        run: kind load docker-image --name tgw trino-gateway:10-SNAPSHOT-amd64 && kind load docker-image --name tgw trino-gateway:10-SNAPSHOT-arm64
```

Comment:
how do we update the version here all the time? Can we make it generic to the current build somehow?

Reply:
Or use some sort of variable